system-int-test/test/cmr/system_int_test/search/variable/collection_measurement_search_test.clj

```diff
@@ -115,4 +118,110 @@
         [coll1 coll2] "measurement1" {:ignore-case true}
 
         "ignore-case false"
-        [] "measurement1" {:ignore-case false}))))
+        [] "measurement1" {:ignore-case false}))
+
+
+    (testing "search collections by variables-h"
+      (are3 [items params options]
+        (let [search-params (merge {:variables-h params}
+                                   (when options
+                                     {"options[variables-h]" options}))]
+          (d/refs-match? items (search/find-refs :collection search-params)))
+
+        "single measurement search"
+        [coll1 coll2]
+        {:0 {:measurement "Measurement1"}}
+        {}
+
+        "no matching measurement"
+        []
+        {:0 {:measurement "Measurement3"}}
+        {}
+
+        "multiple measurements"
+        [coll2]
+        {:0 {:measurement "Measurement1"}
+         :1 {:measurement "Measurement2"}}
+        {}
+
+        "multiple measurements option OR true"
+        [coll1 coll2 coll3 coll4]
+        {:0 {:measurement "Measurement1"}
+         :1 {:measurement "Measurement2"}}
+        {:or true}
+
+        "multiple measurements option OR false"
+        [coll2]
+        {:0 {:measurement "Measurement1"}
+         :1 {:measurement "Measurement2"}}
+        {:or false}
+
+        "single variable search"
+        [coll1 coll2]
+        {:0 {:variable "Variable1"}}
+        {}
+
+        "no matching variable"
+        []
+        {:0 {:variable "Variable3"}}
+        {}
+
+        "multiple variables"
+        [coll2]
+        {:0 {:variable "Variable1"}
+         :1 {:variable "Variable2"}}
+        {}
+
+        "multiple variables option OR true"
```

Comment:
We do have an OR test - I wonder why that typo didn't cause these tests to fail.